### ISSUE TITLE
Create `--orphan` "tng" branch for mastering new floxpkgs template

### ISSUE THREAD

**Issue by limeytexan** [flox/floxpkgs-template#2]
This issue documents the task of creating a new template for creating `floxpkgs` repositories for use with the capacitor. It is expected that we will create this branch using the `git checkout --orphan` flag and that it will have a distinct revision history from that of the current `master` branch.

Notably it is expected that customers will need to rebase against this template over time as we make updates to the capacitor requiring changes to the way it is called from the `floxpkgs` repositories. In an ideal world this would never happen, but in practice we encountered this need several times in the development of the original `flox-lib`.

### Tasklist

- [x] create `tng` branch
- [x] populate new branch with prototype `floxpkgs` repository
    - [ ] including the importing of the `flox` channel by default (so that we can use our channel as a means of publishing updates, experimental versions, tools/utilities, etc.)
    - [x] tapping into all the functionality offered by the capacitor, either directly (probably better) or by way of the `flox` channel flake
    - [ ] include advice / templated example for building software from the user's own organization (should be on the lines of "include Nix expression in template repo, then fork example repository to your organization to make it work")

**Comment by limeytexan** [comment]
We had created the new `tng` branch from the master branch so I moved that aside and recreated it with `--orphan` so that it now has a completely distinct history from master.

I also set the `tng` branch as the default branch so that users cloning the template will get the right branch.

We still need to come up with documentation and workflows for working with floxpkgs.

**Comment by limeytexan** [comment]
Changed this to P1 and changing status to "on hold" because floxpkgs including floxpkgs functionality is waiting on Silvan's efforts to extract the pure version of flox-lib "merge" functionality.

But again, once we can support this, we'll need to document all the various workflows for working with floxpkgs.

**Comment by workingrush** [comment]
depends on https://github.com/flox/capacitor/issues/21
